config-connector/solutions/iam/helm/subnet/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+name: subnet
+version: 0.1.0
+description: Create a subnet and grant a specific member a role to a member for the subnet
```

Comment:
Nit: Create a subnet and grant `a specific member a role` for the subnet

Reply:
Done.

Reply:
Done.